docs/configuration.md

```diff
@@ -257,13 +257,12 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
-  <td><code>spark.files.userClassPathFirst</code></td>
+  <td><code>spark.executor.userClassPathFirst</code></td>
```

Comment:
I was gonna do that but then I noticed none of the other options (e.g. `spark.driver.extraClassPath`) are actually documented anywhere... since they're all pretty much the same (only the context changes), I could reorganize the documentation a bit so that there's this section that lists options that exist for driver and executors, and then add this one, but it's starting to feel like a separate change at this point.